### ISSUE TITLE
SDK Release Plan Target Date data integrity issues between Release Planner & ADO 

### ISSUE THREAD

**Issue by ladonnaq** [Azure/azure-sdk-tools#8871]
Issue: The SDK release target date in release plan does not match date in the ADO release plan work item.  The formatting in ADO is also not correct.  The date needs to be included so we know which year. 

Release plan link: https://aka.ms/sdk-release-planner?release-plan-id=4506e2ba-9156-ef11-a317-000d3a3783ac
Screenshot below 
![image](https://github.com/user-attachments/assets/c5e39535-933c-4c89-9562-829884f9a423)

ADO release plan work item link: https://dev.azure.com/azure-sdk/Release/_queries/edit/23567
Screenshot below 
![image](https://github.com/user-attachments/assets/4a64f243-3678-4289-beda-96305399ed97)

**Comment by maririos** [comment]
@ladonnaq do you have other examples? the one you are mentioning was manually modified by you. Automation correctly set the date:
![image](https://github.com/user-attachments/assets/de901212-54c2-4f0b-b067-759a4c7bec3f)

**Comment by ladonnaq** [comment]
Something must have been corrupted when I exported and imported the work items to change add values of isAPEX & isInitial fields. I will omit any date fields when I export and import in the future. Excel tries to convert anything resembling a date to a "date field" and ADO date fields are not standard (remember Praveen had to just use text field).
Is there a way to push the data for that field back out to the release plans? If not, I can go to each release plan that is showing incorrect and edit in the release plan. That does work as I just tried it.
